scanapp/tests/test_tasks.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
+from django.core.files import File
+import json
```

Comment:
Typically the `json` import which is stdlib should come before django imports that are from a third-party